adafruit_ble_berrymed_pulse_oximeter/adafruit_ble_transparent_uart.py

```diff
@@ -44,7 +48,7 @@ class TransparentUARTService(Service):
         buffer_size=64,
     )
 
-    def __init__(self, service=None):
+    def __init__(self, service: Union[Service, None] = None):
```

Comment:
Whenever `None` is allowed, you can actually use Optional to denote that:

```suggestion
def __init__(self, service: Optional[Service] = None) -> None:
```

In fact it's equivalent to `Union[XYZ, None]`!

Reply:
ohhh that's very nice! will fix (everywhere)